tests/transport_test.go

```diff
@@ -71,25 +84,37 @@ var _ = Describe("Transport Tests", func() {
 			pvcAnn[controller.AnnSecret] = sec.Name
 		}
 
-		By(fmt.Sprintf("Creating PVC with endpoint annotation %q", ep))
+		By(fmt.Sprintf("Creating PVC with endpoint annotation %q", ip))
```

Comment:
do you want to show the filename part of the ep too?

Reply:
yep, np